src/ClamAV.php

```diff
@@ -41,46 +49,27 @@ class ClamAV
      *
      * @var string
```

Comment:
In some places you removed the type hint in favour of the explicit type, should we perhaps treat that consistently everywhere?